core/src/main/java/com/linecorp/armeria/client/DefaultClientRequestContext.java

```diff
@@ -70,7 +72,7 @@ public DefaultClientRequestContext(
             SessionProtocol sessionProtocol, Endpoint endpoint,
             HttpMethod method, String path, @Nullable String query, @Nullable String fragment,
             ClientOptions options, Object request) {
-
+        
```

Comment:
nit: checkstyle error: trailing spaces [RegexpSingleline]

Reply:
Thank you~!!